### PR TITLE
chore: partial jest migration

### PR DESCRIPTION
**Pull request by g-cheishvili**
## Related Issue(s)

closes

## Description

Migrated 102 projects to jest. modules migrated are:
- platform-action-button-group
- core-resizable-card-layout
- platform-split-menu-button
- platform-upload-collection
- platform-value-help-dialog
- core-dynamic-side-content
- platform-object-attribute
- platform-wizard-generator
- core-illustrated-message
- core-vertical-navigation
- platform-message-popover
- core-micro-process-flow
- core-progress-indicator
- core-object-identifier
- core-upload-collection
- platform-object-marker
- platform-object-status
- core-rating-indicator
- core-status-indicator
- core-content-density
- core-datetime-picker
- core-infinite-scroll
- core-side-navigation
- platform-menu-button
- platform-page-footer
- core-busy-indicator
- core-product-switch
- platform-action-bar
- platform-feed-input
- platform-info-label
- core-message-strip
- core-message-toast
- core-object-marker
- core-object-number
- core-object-status
- cx-side-navigation
- platform-thumbnail
- core-dynamic-page
- core-layout-panel
- core-message-page
- core-notification
- core-split-button
- core-date-picker
- core-inline-help
- core-input-group
- core-layout-grid
- core-mobile-mode
- core-time-picker
- cdk-data-source
- core-action-bar
- core-feed-input
- core-info-label
- core-pagination
- core-quick-view
- core-scroll-spy
- core-step-input
- platform-button
- platform-shared
- core-grid-list
- core-scrollbar
- core-calendar
- core-carousel
- core-checkbox
- core-datetime
- core-shellbar
- core-skeleton
- core-splitter
- core-timeline
- platform-link
- platform-list
- platform-menu
- core-popover
- core-theming
- core-avatar
- core-button
- core-facets
- core-select
- core-shared
- core-slider
- core-switch
- core-alert
- core-panel
- core-radio
- core-tests
- core-title
- core-token
- core-utils
- cdk-forms
- core-card
- core-form
- core-icon
- core-link
- core-list
- core-text
- core-tile
- core-time
- core-tree
- datetime-adapter
- core-bar
- moment-adapter
- nx-plugin
- i18n

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
   "packageManager": "yarn@1.22.18",
   "scripts": {
     "preinstall": "npx only-allow yarn",
+    "migrate": "nx g @fundamental-ngx/nx-plugin:migrate-to-jest --project=core-action-bar",
```
Comment (mikerodonnell89) [inline package.json:19]: What is this for?

File: package.json [hunk 2]
```diff
@@ -142,6 +143,8 @@
     "jasmine-spec-reporter": "7.0.0",
     "jest": "29.4.3",
     "jest-environment-jsdom": "29.4.3",
+    "jest-extended": "3.2.4",
+    "jest-preset-angular": "13.1.0",
     "karma": "6.4.1",
```
Comment (mikerodonnell89) [inline package.json:148]: Do we will need karma dependencies?

**Comment by mikerodonnell89** [comment]
So with this PR we are running some tests with karma and others with jest?

**Comment by g-cheishvili** [comment]
> So with this PR we are running some tests with karma and others with jest?

Yes, majority on Jest and few ones on Karma, till we get back to board and update those tests

File: package.json (hunk 1 above)
Comment (g-cheishvili) [reply package.json]: Forgot to remove, it was for quickly testing [auto migration generator](https://github.com/SAP/fundamental-ngx/issues/9813#issuecomment-1540738387)

**Comment by N1XUS** [comment]
@g-cheishvili amazing job! I see a lot of errors (that are not affecting the test results, yet, pollute the logs). Majority of them related to `innerText` property. Can we have a separate task with the list of components that are using that property and review those components on why they actually do that?

**Review by N1XUS — APPROVED** [review]

File: .github/workflows/on-pull-request.yml [hunk 3]
```diff
@@ -62,7 +62,7 @@ jobs:
             - uses: wagoid/commitlint-github-action@v3
               name: Commit lint
 
-            - name: Run Build and Test
+            - name: Run Build and Lint
```
[the remark this answers is not in the document]
Comment (g-cheishvili) [reply .github/workflows/on-pull-request.yml]: Accidental, pushed fixed

**Review by InnaAtanasova — APPROVED** [review]

**Review by InnaAtanasova — APPROVED** [review]

**Pull request merged**